Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutlinerListModel.cpp

```diff
@@ -532,6 +537,12 @@ namespace AzToolsFramework
             break;
         }
 
+        AZ::EntityId entityId = GetEntityFromIndex(index);
+        if (!m_focusModeInterface->IsInFocusSubTree(entityId))
+        {
+            itemFlags &= !Qt::ItemIsEnabled;
```

Comment:
Just double checking: focus mode does a model reset or otherwise invalidates the indices, right?

Reply:
Not right now. I may have to do it in a follow-up PR since we need focus mode to control the expanded state of containers in the Outliner.

Reply:
OK, just be careful here if you don't end up doing so; you're in undefined behavior territory when state gets changed without a notification (though I suspect hover will specifically always work since data gets requeried by the view)

Reply:
Yeah I was honestly surprised that is updates correctly. Sounds like the flags get refreshed automatically.